src/ioctls/system.rs

```diff
@@ -239,6 +239,32 @@ impl Kvm {
         }
     }
 
+    /// Returns the "disable idle exiting" capability.
+    ///
+    /// The `KVM_CAP_X86_DISABLE_EXITS` capability provides userspace with per-VM capability
+    /// to not intercept MWAIT/HLT/PAUSE/CSTATE, which means though instructions won't cause
+    /// vm-exits and helps to improve latency in some workloads.
+    ///
+    /// # Example
+    ///
+    /// ```
+    /// # extern crate kvm_bindings;
+    /// # use kvm_bindings::{KVM_X86_DISABLE_EXITS_HLT, KVM_X86_DISABLE_EXITS_PAUSE};
+    /// # use kvm_ioctls::Kvm;
+    /// let kvm = Kvm::new().unwrap();
+    /// assert!(kvm.get_disable_exits() & KVM_X86_DISABLE_EXITS_HLT != 0);
+    /// assert!(kvm.get_disable_exits() & KVM_X86_DISABLE_EXITS_PAUSE != 0);
+    /// ```
+    #[cfg(any(target_arch = "x86", target_arch = "x86_64"))]
+    pub fn get_disable_exits(&self) -> u32 {
+        let x = self.check_extension_int(Cap::DisableExits);
+        if x > 0 {
+            x as u32
+        } else {
+            0
+        }
```

Comment:
I think this can be slightly improved.
```suggestion
        u32::try_from(x).unwrap_or(0)
```